apstools/plans/alignment.py

```diff
@@ -34,8 +40,14 @@
 
 def lineup(
     # fmt: off
-    detectors, axis, minus, plus, npts,
-    time_s=0.1, peak_factor=4, width_factor=0.8,
+    detectors,
```

Comment:
Why is this reformatted?  It is in a `fmt: off` section.

Reply:
Changed back, I formatted the whole file. Didn't realize that is what fmt: off mean